src/app/groups/specific-group/plans-display/plans-display.component.html

```diff
@@ -20,7 +21,7 @@
           </td>
         </ng-container>
 
-        <ng-container matColumnDef="Create by">
+        <ng-container matColumnDef="Creator">
```

Comment:
Volodymyr told that "Created by" is more properly

Reply:
It's internal, not visible for users  column definition so I used this word